dipy/segment/tests/test_clustering.py

```diff
@@ -302,7 +323,8 @@ def test_cluster_map_remove_cluster():
 
     clusters.remove_cluster(cluster2)
     assert_equal(len(clusters), 2)
-    assert_array_equal(list(itertools.chain(*clusters)), list(itertools.chain(*[cluster1, cluster3])))
+    assert_array_equal(list(itertools.chain(*clusters)),
+                       list(itertools.chain(*[cluster1, cluster3])))
```

Comment:
Like this! Thanks - this one is exactly the way the other ones about should also be.